pages/common/code.md

```diff
@@ -17,3 +17,7 @@
 - Open a file or directory in the currently open VS Code window:
 
 `code --reuse-window {{path/to/file_or_folder}}`
+
+- Compare two files in VS Code:
+
+`code -d {file1} {file2}`
```

Comment:
Could you add an extra layer of `{}`, like this please?

```suggestion
`code -d {{file1}} {{file2}}`
```

Reply:
Sure